app/assets/stylesheets/application.scss

```diff
@@ -27,4 +29,43 @@ ul, li {
 }
 .clear-fix:after {
   clear: both;
+}
+
+.bs-callout {
+  padding: 20px;
```

Comment:
Properties should be ordered border, border-left-width, border-radius, margin, padding